validator/java/pom.xml

```diff
@@ -19,16 +19,16 @@
         <validator.proto.directory>${project.basedir}/src/main/proto</validator.proto.directory>
 
         <!-- library versions -->
-        <build-helper-maven-plugin.version>3.0.0</build-helper-maven-plugin.version>
+      <build-helper-maven-plugin.version>3.1.0</build-helper-maven-plugin.version>
```

Comment:
Seems like the indentation is broken here and everywhere else. Can you fix?

Reply:
yeah, I can do that, this will obfuscate the material changes, but now that you've seen them it's not a worry

Reply:
It's possible to [hide whitespaces from the diffs](https://github.com/ampproject/amphtml/pull/27951/files?w=1) by adding `?w=1` to the Github URL. Still looks good!